bitnami/influxdb/templates/influxdb/deployment-standalone.yaml

```diff
@@ -64,16 +64,28 @@ spec:
           imagePullPolicy: {{ .Values.volumePermissions.image.pullPolicy | quote }}
           command:
             - /bin/bash
-            - -c
+            - -cx
```

Comment:
Why do you add `-x`? It's for tracing, right ? See https://www.gnu.org/software/bash/manual/html_node/The-Set-Builtin.html
I wouldn't add it by default

Reply:
I had actually removed that based on feedback but I made a mistake when rebasing, sorry.